### PR TITLE
Spike OC

### PR DESCRIPTION
**Pull request by mattiaerre**
# Description

This PR is a spike that shows how to integrate an [OC](https://github.com/opentable/oc) component into a Rect/Redux app.

- new `oc-date-time-now` component (published to the oc-registry.herokuapp.com registry)

- new React component that wraps the `oc-component` html tag

- new React action creator that triggers `oc.renderUnloadedComponents()`

- use `bullet-pubsub` in order to enable OC-React and React-OC communication

Live example: [https://supersonic-lobster.herokuapp.com/](https://supersonic-lobster.herokuapp.com/)

💡 hint: watch the dev console

Kanban board's [card](https://github.com/mattiaerre/supersonic-lobster/projects/1#card-1692503)

### CODE REVIEW

**Comment by mattiaerre** [comment]
// cc @matteofigus @nickbalestra

**Review by matteofigus — APPROVED** [review]

**Comment by mattiaerre** [comment]
It really works 😮 

OC

https://github.com/mattiaerre/supersonic-lobster/blob/06ce1e784557a694706d9bd2e94ae75307f9223c/components/oc-date-time-now/src/index.js#L14-L36

React

https://github.com/mattiaerre/supersonic-lobster/blob/06ce1e784557a694706d9bd2e94ae75307f9223c/src/index.js#L10-L37

// cc @matteofigus @nickbalestra

File: components/oc-date-time-now/public/index.js
Comment (matteofigus) [inline components/oc-date-time-now/public/index.js]: do you think we could reuse oc.events? https://github.com/opentable/oc/wiki/Browser-client#oceventsoneventname-callback

Comment (mattiaerre) [reply components/oc-date-time-now/public/index.js]: Even better as it will be one less dependency we have to use. I only need to make sure it is available for the React app as well. I'll try it later.

**Comment by mattiaerre** [comment]
as per @matteofigus 's feedback now using `oc.events`

// @nickbalestra

**Comment by mattiaerre** [comment]
// cc @scottlepich @mdboop

**Comment by mattiaerre** [comment]
Hi @matteofigus @nickbalestra @scottlepich @mdboop
I'd really like to have your opinion about this PR before I merge it
I ❤️  you ('cause it's V-day isn't it 😅 )

**Comment by mattiaerre** [comment]
#TIL: nobody loves me 💔 

// cc @nickbalestra @scottlepich @mdboop

**Comment by matteofigus** [comment]
Sorry for being nobody 💔

**Comment by mattiaerre** [comment]
that was not for you @matteofigus but I really would like to have another opinion, point of view about this 😅 

// cc @nickbalestra @scottlepich @mdboop @chriscartlidge

**Comment by mattiaerre** [comment]
#YOLOMERGE

**Pull request merged**
